node-services/sharing-service/routes/index.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express');
+const fetch = require('node-fetch');
+const shareReportWithOthers = require('../controllers/sharedWithOthers');
+const shareReportWithMe = require('../controllers/reportSharedWithMe');
+
+const router = express.Router();
+
+router.use(express.json());
+router.use(express.urlencoded({ extended: false }));
+
+// responseCode -1 Empty email parameter
+// responseCode  0 User doesn't exist
+// responseCode  1 Report shared successfully
+
+router.post('/', async (req, res) => {
```

Comment:
let's not keep it default route, give some meaningful path

Reply:
I have provided that path name while using this route in index file